### ISSUE TITLE
ReactiveCommand is changed?

### ISSUE THREAD

**Issue by neslekkim** [RolandPheasant/Dynamic.Trader#8]
Trying to learn, and downloaded this code, but after upgrading reactive and stuff, I get error in LogEntryViewer, that the ReactiveCommand is defined wrong?

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1061	'ReactiveCommand' does not contain a definition for 'Dispose' and no accessible extension method 'Dispose' accepting a first argument of type 'ReactiveCommand' could be found (are you missing a using directive or an assembly reference?)	Trader.Client	LogEntryViewer.cs	91	Active
Error	CS0029	Cannot implicitly convert type 'ReactiveUI.ReactiveCommand<System.Reactive.Unit, System.Reactive.Unit>' to 'ReactiveUI.ReactiveCommand'	Trader.Client	LogEntryViewer.cs	77	Active
Error	CS0718	'ReactiveCommand': static types cannot be used as type arguments	Trader.Client	LogEntryView.xaml.cs	26	Active
Error	CS0722	'ReactiveCommand': static types cannot be used as return types	Trader.Client	LogEntryViewer.cs	27	Active

Since I'm trying to learn this, I don't know what the error is though.. *blush*

**Comment by vector-man** [comment]
Yeah, after a Nuget update, this broke my project: 'ReactiveCommand': static types cannot be used as type arguments. Any ideas?

**Comment by worldbeater** [comment]
[ReactiveCommand abstract base class has been deprecated and removed](https://reactiveui.net/blog/2018/11/reactive-command-abstract) in latest ReactiveUI releases, so need to change all occurrences of `ReactiveCommand` to `ReactiveCommand<Unit, Unit>` during the upgrade.
